crypto/mem_clr.c

```diff
@@ -60,6 +60,7 @@
 #include <string.h>
 #include <openssl/crypto.h>
 
+extern unsigned char cleanse_ctr;
```

Comment:
FWIW I implemented an alternative `OPENSSL_cleanse()` that doesn't need the `cleanse_ctr` variable https://github.com/ghedo/openssl/commit/ef8adf8fad217ea128d8d8726e091df983784169. Should I open a pull request?